README.md

```diff
@@ -1,66 +1,112 @@
-   - [ ] Generating performance benchmark graphs, auto-publishing to GitHub Pages
-   - [x] Documentation of API docs and [mdBook](https://github.com/rust-lang/mdBook) using GitHub Pages
-   - [x] Linted commit messages with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) 
-   - [x] Semantic version bumping, `CHANGELOG.md` updates, and new package releases
-
-## Configure
-| Tool                     | File path                                                | Reference                                                                                                        |
-|--------------------------|----------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
-| GitHub Codespaces        | [`devcontainer.json`](./.devcontainer/devcontainer.json) | [Reference](https://containers.dev/implementors/json_reference/)                                                 |
-| GitHub Actions           | [`.github/workflows`](./.github/workflows)               | [Reference](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions)               |
-| Cargo package            | [`Cargo.toml` ](./Cargo.toml)                            | [Reference](https://doc.rust-lang.org/cargo/reference/manifest.html)                                             |
-| Clippy (Rust linter)     | [`.clippy.toml`](./.clippy.toml)                         | [Repository](https://github.com/rust-lang/rust-clippy), [ Reference ]( https://rust-lang.github.io/rust-clippy/) |
-| Rustfmt (Rust formatter) | [`.rustfmt.toml`](./.rustfmt.toml)                       | [Repository](https://github.com/rust-lang/rustfmt), [ Reference](https://rust-lang.github.io/rustfmt/)           |
-| Commitlint               | [`.commitlintrc.json`](./.commitlintrc.json)            | [Repository](https://github.com/conventional-changelog/commitlint), [Reference](https://commitlint.js.org/#/)    |
-| `cargo-deny`             | [`deny.toml`](./deny.toml)                               | [Repository](https://github.com/EmbarkStudios/cargo-deny)                                                        |
-
-## Run scripts locally
- - Run unit/integration/doc tests: `cargo test`
- - Run fuzz tests: `cargo fuzz <fuzz-target>`
- - Run Rustfmt: `cargo fmt`
- - Run Clippy: `cargo clippy`
- - Run performance benchmarks: `cargo bench`
- - Generate API docs for crate: `cargo doc`
- - Generate mdBook docs for crate: `mdbook build`
- - Run security audits: `cargo audit` (requires installing [`cargo-audit`](https://crates.io/crates/cargo-audit) locally)
- - Lint commit messages: `npm run lint:commit` (requires installing [`commitlint`](https://commitlint.js.org/#/) locally with `npm install`)
-
-----
-
-# {{library}}
-{{library description}}
+# Differ.rs
```

Comment:
Can you add some README badges after the h1 header? Badges are really helpful since they're one of the first things a reader sees, and one of the quickest ways to gain some information about the repo. I recommend this:
```markdown
# Differ.rs
[![License](https://img.shields.io/badge/license-MIT%20%26%20Apache%202.0-green)](#license)
[![CI](https://github.com/nlp-rs/differ.rs/actions/workflows/main.yml/badge.svg)](https://github.com/nlp-rs/differ.rs/actions/workflows/main.yml)
[![Security audit](https://github.com/nlp-rs/differ.rs/actions/workflows/security-audit.yml/badge.svg)](https://github.com/nlp-rs/differ.rs/actions/workflows/security-audit.yml)
```